csrc/transform_view.cpp

```diff
@@ -277,23 +271,13 @@ class SplitTransform final : public ViewTransform {
         "Didn't expect to apply view transformations on an iter domain",
         " starting at a non-zero position.");
 
-    Val* remainder = ceilDiv(id->extent(), factor);
-
-    // outer loop IterDomain
-    IterDomain* factor_id =
-        IterDomainBuilder(FusionGuard::getCurFusion()->zeroVal(), factor)
-            .parallel_type(id->getParallelType())
-            .iter_type(id->getIterType())
-            .is_rfactor_domain(true)
-            .build();
-
-    // inner loop IterDomain
-    IterDomain* remainder_id =
-        IterDomainBuilder(FusionGuard::getCurFusion()->zeroVal(), remainder)
-            .is_rfactor_domain(true)
-            .build();
-
-    IrBuilder::create<Split>(factor_id, remainder_id, id, factor, false);
+    auto [factor_id, remainder_id] = IterDomain::split(
+        id,
+        factor,
+        /*inner_split*/ false,
+        /*start_offset*/ nullptr,
+        /*stop_offset*/ nullptr,
+        /*rfactor_domain*/ true);
```

Comment:
https://clang.llvm.org/extra/clang-tidy/checks/bugprone/argument-comment.html

`/*param_name=*/` has the advantage of being checked by clang-tidy. I'm not sure `/*param_name*/` has the same effect. Please ignore if it does. 

```suggestion
        /*inner_split=*/false,
        /*start_offset=*/nullptr,
        /*stop_offset=*/nullptr,
        /*rfactor_domain=*/true);
```

Reply:
That's good to know! Thanks.